src/main/java/com/simibubi/create/foundation/events/ClientEvents.java

```diff
@@ -322,17 +322,14 @@ public static void getFogDensity(ViewportEvent.RenderFog event) {
 
 		ItemStack divingHelmet = DivingHelmetItem.getWornItem(entity);
 		if (!divingHelmet.isEmpty()) {
-			if (FluidHelper.isWater(fluid)) {
+			if (FluidHelper.isWater(fluid) && AllConfigs.server().equipment.shouldScaleWaterFogDistance.get()) {
```

Comment:
I don’t think this is a very elegant way of solving your issue, as the diving helmet’s increased visibility will now not work in normal ocean biomes if this config is set.

Why is this a server config?

Reply:
I chose the config server, because the range of visibility is a gameplay element and an item property. I acted by analogy that if the subject had night vision, for example, I would take it to the config server. 
If required, I will put it in the client config. But then in multiplayer, each player will have to turn off this property on their own.

Reply:
No that’s correct reasoning; but it should just be a biome tag instead as TropheusJ indicated.